stk/calculators/optimization/optimizers.py

```diff
@@ -648,102 +650,503 @@ def optimize(self, mol, conformer=-1):
+        Solvent to use in GBSA implicit solvation method.
+        For details see
+        https://xtb-docs.readthedocs.io/en/latest/gbsa.html.
+
+    solvent_grid : :class:`str`
+        Grid level to use in SASA calculations for GBSA implicit
+        solvent.
+        Can be one of ``'normal'``, ``'tight'``, ``'verytight'``
+        or ``'extreme'``.
+        For details see
+        https://xtb-docs.readthedocs.io/en/latest/gbsa.html.
+
+    charge : :class:`str`
+        Formal molecular charge.
+
+    num_unpaired_electrons : :class:`str`
+        Number of unpaired electrons.
+
+    unlimited_memory : :class:`bool`
+        If ``True`` :meth:`optimize` will be run without constraints on
+        the stack size. If memory issues are encountered, this should
+        be ``True``, however this may raise issues on clusters.
 
+    incomplete : :class:`set`
+        A :class:`set` holding tuples of the form ``(mol, conformer)``,
+        which are the :class:`.Molecule` objects and conformer ids
+        passed to :meth:`optimize` whose optimzation was incomplete.
+
+    Examples
+    --------
     Note that for :class:`.MacroMolecule` objects assembled by ``stk``
-    :class:`GFNXTB` should usually be used in a
-    :class:`OptimizerSequence`. This is because GFN-xTB only uses
+    :class:`XTB` should usually be used in a
+    :class:`OptimizerSequence`. This is because xTB only uses
     xyz coordinates as input and so will not recognize the long bonds
     created during assembly. An optimizer which can minimize
-    these bonds should be used before :class:`GFNXTB`.
+    these bonds should be used before :class:`XTB`.
 
     .. code-block:: python
 
         bb1 = StructUnit2.smiles_init('NCCNCCN', ['amine'])
         bb2 = StructUnit2.smiles_init('O=CCCC=O', ['aldehyde'])
         polymer = Polymer([bb1, bb2], Linear("AB", [0, 0], 3))
 
-        gfnxtb = OptimizerSequence(
+        xtb = OptimizerSequence(
             UFF(),
-            GFNXTB('/opt/gfnxtb/xtb')
+            XTB(xtb_path='/opt/gfnxtb/xtb', unlimited_memory=True)
+        )
+        xtb.optimize(polymer)
+
+    By default, all optimizations with xTB are performed using the
+    ``--ohess`` flag, which forces the calculation of a numerical
+    Hessian, thermodynamic properties and vibrational frequencies.
+    :meth:`optimize` will check that the structure is appropriately
+    optimized (i.e. convergence is obtained and no negative vibrational
+    frequencies are present) and continue optimizing a structure (up to
+    :attr:`max_runs` times) until this is achieved. This loop, by
+    default, will be performed at the same :attr:`opt_level`. The
+    following examples shows how a user may optimize structures with
+    tigher convergence criteria (i.e. different :attr:`opt_level`)
+    until the structure is sufficiently optimized. Furthermore, the
+    calculation of the Hessian can be turned off using
+    :attr:`max_runs` = 1 and :attr:`calculate_hessian`=False.
```

Comment:
This needs to be written as a full sentence.

```
Furthermore, the calculation of the Hessian can be turned off by setting :attr:`max_runs` to ``1`` and 
:attr:`calculate_hessian` to ``False``.

```

This is because the syntax highlight will be off in the way you have written it. (For example no backticks around the literal values of 1 and False but also you cant mix hyperlinks, such as the attr names, with literal values such as the 1 and False. Therefore writing it in a full sentence is the best way.